admin_guide/crictl.adoc

```diff
@@ -0,0 +1,466 @@
+    }
+  },
+  "version": {
+    "version": "1.10.7-dev"
+  }
+}
+----
+
+==== Stopping Pods
+We can stop a specific pod using the `*crictl stopp*` command. Since *oc* defaults to restarting a pod when stopped or removed, we can see that a new pod was started below.
+----
+4# crictl pods
+POD ID              CREATED             STATE               NAME                NAMESPACE           ATTEMPT
+1f1c536a3e819       42 minutes ago      SANDBOX_READY       my-pod              default             0
+
+# crictl stopp 1f1c536a3e819
+Stopped sandbox 1f1c536a3e819
+
+# crictl pods
+POD ID              CREATED                  STATE               NAME                NAMESPACE           ATTEMPT
+5832b44f04ba4       Less than a second ago   SANDBOX_READY       my-pod              default             1
+1f1c536a3e819       42 minutes ago           SANDBOX_NOTREADY    my-pod              default             0
+----
+
+==== Deleting Pods
+Stopped pods can be deleted using the `*crictl rmp*` command.
+----
+# crictl rmp 1f1c536a3e819
+Removed sandbox 1f1c536a3e819
+
+# crictl pods
+POD ID              CREATED             STATE               NAME                NAMESPACE           ATTEMPT
+5832b44f04ba4       18 seconds ago      SANDBOX_READY       my-pod              default             1
+----
+
+=== Containers
+
+==== Listing Containers
+We can list the containers using the `*crictl ps*` command.
+----
+# crictl ps
+CONTAINER ID        IMAGE                                                              CREATED             STATE               NAME                ATTEMPT
+95b8d9c69dd32       49f7960eb7e4cb46f1a02c1f8174c6fac07ebf1eb6d8deffbcb5c695f1c9edd5   38 seconds ago      CONTAINER_RUNNING   shell               0
+5b110a8344d3a       73a557ff177a50bab42b5e36c9c8ba4c9fa18f2fe7a74f04900856ab1648cb89   38 seconds ago      CONTAINER_RUNNING   my-httpd            0
+----
+
+By default, `*crictl ps*` only lists the running containers, to view all the containers in store use the *--all/-a* flag. As you can see below, a third container that is in the _created_ state showed up.
+----
+# crictl ps -a
+CONTAINER ID        IMAGE                                                              CREATED             STATE               NAME                ATTEMPT
+e578fef6bff53       quay.io/crio/redis:alpine                                          5 seconds ago       CONTAINER_CREATED   podsandbox1-redis   0
+95b8d9c69dd32       49f7960eb7e4cb46f1a02c1f8174c6fac07ebf1eb6d8deffbcb5c695f1c9edd5   6 minutes ago       CONTAINER_RUNNING   shell               0
+5b110a8344d3a       73a557ff177a50bab42b5e36c9c8ba4c9fa18f2fe7a74f04900856ab1648cb89   6 minutes ago       CONTAINER_RUNNING   my-httpd            0
+----
+
+==== Executing Processes within Containers
+We can use the `*crictl exec*` command to run processes inside a running container. This can be used for in-depth inspection of running containers.
+
+If we just want to run a single process, we can use the *--sync* flag.
```

Comment:
--sync is for when we expect the process to run to completion.

Reply:
Fixed